src/main/java/frc/robot/Config.java

```diff
@@ -248,4 +252,65 @@ public static final class AutoConstants {
         new TrapezoidProfile.Constraints(
             kMaxAngularSpeedRadiansPerSecond, kMaxAngularSpeedRadiansPerSecondSquared);
   }
+  public class ArmConfig {
+    public static final boolean SET_INVERTED = true;
+
+    public static final int CURRENT_LIMIT = 60;
+
+    //soft limit constant for bottom arm
+    public static final float arm_forward_limit = (float)Math.toRadians(135);
+    public static final float arm_reverse_limit = (float)Math.toRadians(40);
```

Comment:
These constants define the soft limits which tell the sparkmax what angles the sparkmax will allow. 

You also have constants down below called `MIN_ARM_ANGLE_DEG` and `MAX_ARM_ANGLE_DEG` that are used to clamp the setpoint. There is a bit of mismatching between the values of each. The setpoint is clamped between 0 and 95 right now, but the soft limits won't let the sparkmax go below 40 degrees.

Reply:
Replace 40 and 135 by constants MIN_ARM_ANGLE_DEG and MAX_ARM_ANGLE_DEG